vector/src/main/java/im/vector/activity/VectorMemberDetailsActivity.java

```diff
@@ -130,9 +130,10 @@
     private VectorMemberDetailsDevicesAdapter mDevicesListViewAdapter;
 
     // UI widgets
-    @BindView(R.id.avatar_img)
+    @BindView(R.id.big_avatar_img)
     ImageView mMemberAvatarImageView;
-    @BindView(R.id.member_details_presence)
+    // Tchap: hide the user presence until the actual information is available
+//    @BindView(R.id.member_details_presence)
```

Comment:
mPresenceTextView will always be null then. Is it the expected thing? Setting visibility to Gone in the layout (has you have done) is enough

Reply:
yes this is the expected thing. I want to keep `mPresenceTextView` null to prevent us from updating it